modules/ghlint-rules/src/main/kotlin/net/twisterrob/ghlint/rules/InvalidExpressionUsageRule.kt

```diff
@@ -25,8 +25,8 @@ public class NeverExpressionRule : VisitorRule, WorkflowVisitor {
 	private companion object {
 		const val uses = "actions/checkout@\${{ github.sha }}"
 
-		val NeverExpression = Issue(
-			id = "NeverExpression",
+		val InvalidExpression = Issue(
+			id = "InvalidExpressionUsage",
```

Comment:
Keep them in sync

Reply:
This should be resolved - must've pushed the commits up separately.

Reply:
still different, I meant

```suggestion
		val InvalidExpressionUsage = Issue(
			id = "InvalidExpressionUsage",
```

did you understand the same?